examples/react-ssr/vite.config.ts

```diff
@@ -94,20 +87,6 @@ export function vitePluginSsrMiddleware({
       return;
     },
 
-    // [feedback] "server" environment full-reload still triggers "client" full-reload?
-    hotUpdate(ctx) {
-      if (ctx.environment.name === "ssr") {
-        const ids = ctx.modules.map((mod) => mod.id).filter(typedBoolean);
-        if (ids.length > 0) {
-          const invalidated = runner.moduleCache.invalidateDepTree(ids);
-          console.log("[ssr:invalidate]", ctx.file);
-          debug("[ssr:handleUpdate]", { ids, invalidated: [...invalidated] });
-          return [];
-        }
-      }
-      return;
-    },
```

Comment:
I thought this was necessary before, but it looks like now channel is isolated and modifying component only reloads server code while keeping client hmr.